src/utils/social-media.ts

```diff
@@ -9,3 +9,27 @@ export const socialMediaUrls = {
   [SocialMedia.twitter]: 'https://twitter.com/bucky_roberts',
   [SocialMedia.youtube]: 'https://www.youtube.com/user/thenewboston',
 };
+
+export const socialMediaHandles = {
+  [SocialMedia.facebook]: 'TheNewBoston',
+  [SocialMedia.github]: 'thenewboston-developers',
+  [SocialMedia.linkedin]: 'thenewboston',
+  [SocialMedia.reddit]: 'r/thenewboston',
+  [SocialMedia.slack]: 't/thenewboston',
+  [SocialMedia.twitter]: '@bucky_roberts',
+  [SocialMedia.youtube]: 'thenewboston',
+};
+
+export const socialMediaDescriptions = {
+  [SocialMedia.facebook]: 'Stay up to date by following us on our Facebook page.',
+  [SocialMedia.github]:
+    'You can find the official repositories for our projects on our GitHub and start contributing today.',
+  [SocialMedia.linkedin]: 'Follow us on LinkedIn and expand your network.',
+  [SocialMedia.reddit]:
+    'Our Reddit community is built to help others learn about computer science and other academic subjects.',
+  [SocialMedia.slack]:
+    'With 2049 members our Slack team is the central communication channel for app ideas, development & design.',
```

Comment:
Would it be more appropriate to be `With over thousands of members...` as the member count is dynamic?